test/integration/config_1/example_spec.rb

```diff
@@ -25,6 +25,7 @@
   its('content') { should match(/bind 0.0.0.0:8081/) }
   its('content') { should match(/bind 0.0.0.0:8180/) }
   its('content') { should match(/backend servers/) }
+  its('content') { should match(/default_backend servers/) }
```

Comment:
It should check for both though that the top level item `backend` is there, and that a default backend is set. I just have to figure out how.